LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 EasyPost
+Copyright (c) 2014- EasyPost
```

Comment:
Nit: let's leave out the trailing dash for consistency

Reply:
Resolved in b058dda5a49aa33e5cce1deaf32223955ba2acb9